src/containers/BooksForm.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const INPUT = styled.input`
+  padding: 7px;
+  margin: 2% 3%;
+  margin-left: -15%;
+`;
+
+const SELECT = styled.select`
+  padding: 5px;
+  background: green;
+  color:white;
+  margin-right: 2%;
+`;
+
+const categories = [
+  'Action',
+  'Biography',
+  'History',
+  'Horror',
+  'Kids',
+  'Learning',
+  'Sci-Fi',
+];
```

Comment:
[Optional] We can add **'All'** to the categories. consider a case where you have to select all the books.